source/PlayState.hx

```diff
@@ -5,13 +5,17 @@ import flixel.FlxState;
 import flixel.graphics.FlxGraphic;
 import flixel.tile.FlxTilemap;
 import flixel.util.FlxColor;
+import flixel.FlxObject;
+import flixel.input.keyboard.FlxKey;
 
 class PlayState extends FlxState
 {
 	public var GRAVITY(default, never):Float = 600;
 	
 	private var map:FlxTilemap;
-	private var player:Player;
+	public var player:Player;
+	private var platform:Platforms;
+	private var x:Float = 0;
```

Comment:
I don't think this variable is being used for anything. If it isn't, then remove it.